assets/src/admin/media-sharing/index.js

```diff
@@ -428,29 +498,55 @@ const MediaSharingApp = ( {
 									/>
 								</div>
 								{ ONEMEDIA_PLUGIN_TAXONOMY_TERM === imageType && (
-									<div className="onemedia-media-edit-button">
-										<Button
-											isSmall
-											variant="secondary"
-											icon="edit"
-											onClick={ ( e ) => {
-												e.stopPropagation();
-												handleEditMedia(
-													media.id,
-													imageType,
-												);
-											} }
-											title={ __( 'Edit Media', 'onemedia' ) }
-										>
-											{ __( 'Edit', 'onemedia' ) }
-										</Button>
-									</div>
+									<>
+										<div className="onemedia-media-edit-button">
+											<Button
+												isSmall
+												variant="secondary"
+												icon="edit"
```

Comment:
Let's import the actual IconType from `@wordpress/icons` instead of passing a string - especially as we don't have any type validation here.